src/components/Wallets/Details/SelectedWalletDetails.tsx

```diff
@@ -7,51 +14,72 @@ import { TokenRowWallets } from "~/components/Tokens/TokenRowWallets";
 import { type TransferredCoinInterface } from "~/routes/app/wallets/interface";
 import Button from "~/components/Atoms/Buttons/Button";
 import IconTrashRed from "@material-design-icons/svg/outlined/delete.svg?jsx";
+import { type Wallet } from "~/interface/auth/Wallet";
+import {
+  SelectedWalletDetailsContext,
+  SelectedWalletNameContext,
+} from "~/routes/app/wallets";
+import { server$ } from "@builder.io/qwik-city";
+import { connectToDB } from "~/database/db";
+import { type Token } from "~/interface/token/Token";
 
 interface SelectedWalletProps {
-  selectedWallet: Signal<WalletTokensBalances | null>;
   chainIdToNetworkName: { [key: string]: string };
   isDeleteModalopen: Signal<boolean>;
   isTransferModalOpen: Signal<boolean>;
   transferredCoin: TransferredCoinInterface;
+  selWallet: Signal<Wallet | null>;
 }
 
+export const fetchAllTokens = server$(async function () {
+  const db = await connectToDB(this.env);
+
+  const tokens = await db.select<Token>("token");
+  return tokens;
+});
+
 export const SelectedWalletDetails = component$<SelectedWalletProps>(
   ({
-    selectedWallet,
     chainIdToNetworkName,
     isDeleteModalopen,
     isTransferModalOpen,
     transferredCoin,
   }) => {
-    if (!selectedWallet.value) return <></>;
-    let shortAddress = selectedWallet.value.wallet.address;
+    const selectedWalletFromContext = useContext(SelectedWalletDetailsContext);
```

Comment:
I would rename `selectedWalletFromContext` to `selectedWalletDetails`

Reply:
fixed